UI/Navigation/NavigationWithXaml/README.md

```diff
@@ -0,0 +1,3 @@
+# Navigate via XAML
+
+NavigateWithXAML is a sample Uno Platform application demonstrating navigating between pages using the `uen:Navigation.Request`.
```

Comment:
```suggestion
**NavigateWithXAML** is a sample Uno Platform application demonstrating how to navigate between pages using the Uno.Extensions Navigation.Request attached property in XAML.
```

We should also point people to the docs: https://platform.uno/docs/articles/external/uno.extensions/doc/Learn/Navigation/HowTo-NavigateInXAML.html